xdebug.c

```diff
@@ -807,15 +807,15 @@ PHP_FUNCTION(xdebug_set_time_limit)
    Dummy function to return original error reporting level when 'eval' has turned it into 0 */
 PHP_FUNCTION(xdebug_error_reporting)
 {
-	if (ZEND_NUM_ARGS() == 0 && XG_BASE(error_reporting_overridden) && xdebug_is_debug_connection_active_for_current_pid()) {
+	if (ZEND_NUM_ARGS() == 0 && XG_BASE(error_reporting_overridden) && xdebug_is_debug_connection_active()) {
 		RETURN_LONG(XG_BASE(error_reporting_override));
 	}
 	XG_BASE(orig_error_reporting_func)(INTERNAL_FUNCTION_PARAM_PASSTHRU);
 }
 /* }}} */
 
 /* {{{ proto void xdebug_pcntl_exec(void)
-   Dummy function to prevent time limit from being set within the script */
+   Dummy function to stop profiling when we run pcntl_exec */
```

Comment:
Can you make this specific change in a separate commit? It's not related to this change. No need to create a separate pull request for it.